Makefile

```diff
@@ -52,14 +52,24 @@ uv: ## Install UV
 ruff: ## Install Ruff
 	@uvx ruff --help &> /dev/null || uv tool install ruff
 
-
 .PHONY: verify
 verify: uv ruff  ## install all required tools
 	@cd $(PY_DIR) && uv lock --check
 	@cd $(PY_DIR) && uvx ruff check --show-fixes
 
-.PHONY: test-python
-test-python: ## Run Python unit test.
-	uv install --dev
-	PYTHONPATH=$(PROJECT_DIR) pytest ./python/kubeflow
+.PHONY: uv-venv
+uv-venv:
+	uv venv
```

Comment:
Shall we check if `.venv` already exists before running `uv venv` ?
IIRC, `uv venv` will always create a new environment, so command will re-install packages all the time.

Reply:
done